src/libstd/time.rs

```diff
@@ -210,6 +210,13 @@ impl Instant {
     }
 }
 
+#[stable(feature = "time2", since = "1.8.0")]
```

Comment:
The version should be 1.28.0, and the feature name could be e.g. `instant_default`.